queue_processors/autoreduction_processor/post_process_admin.py

```diff
@@ -206,12 +214,62 @@ def specify_instrument_directories(self,
         # Specify directories where autoreduction output will go
         return temporary_directory + instrument_output_directory
 
-    def reduction_started(self):
-        """Log and update AMQ message to reduction started"""
-        logger.debug("Calling: %s\n%s",
-                     ACTIVEMQ_SETTINGS.reduction_started,
-                     self.message.serialize(limit_reduction_script=True))
-        self.client.send(ACTIVEMQ_SETTINGS.reduction_started, self.message)
+    @staticmethod
+    def write_and_readability_checks(directory_list, read_write):
```

Comment:
I think this would be cleaner as 2 methods.  e.g. `is_directory_writable(directory)` and `is_directory_readable(directory)` This also would not increase run time as this is already called twice on separate loops.

As a side point, this method is only called within `path_access_validate` and if it ever returns false an exception is raised. It might be nicer to simply raise an exception here and let it propagate up the stack. 

e.g.
```python
def verify_directory_readable(directory):
    if not os.access(directory, os.R_OK):
        raise some error
```
Then in the other method:
```python
for directory in readable_directories:
    verify_directory_readable(directory)
for directory in writable_directories:
    verify_directory_writable(directory)

```